src/cc/lib/benchmark/search_benchmark.cc

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <algorithm>
+#include <iostream>
+#include <numeric>
+#include <random>
+#include <string>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+#ifdef SNARK_PLATFORM_LINUX
+#include <mimalloc-override.h>
+#endif
+
+static void BM_LOWER_BOUND(benchmark::State &state)
+{
+    size_t max_count = state.range(0);
```

Comment:
Where does this use our code? Is it just benchmark on rng?

Reply:
It is how we picked this magic const:
![image](https://github.com/microsoft/DeepGNN/assets/134559/cb13ab05-d967-41d4-a11c-ea86d35dd6b4)

Reply:
I see, maybe a docstring would be helpful here or there for future matinence